vrothberg/vgrep
# v2.2.0

User-visible changes:
 * Changed yellow to magenta to improve readability on light color schemes
   (e.g., tango).
 * Auto-detect VSCode terminal and print line column in a clickable format
   for easy browing.
 * Vgrep has a logo.

Miscellaneous and janitorial changes.
 * update x/sys to latest commit
 * check all errors
 * Dockerfile: pin to golang:1.13
 * update dependencies
 * enable Cirrus CI
 * Makefile: go and tools clean ups
 * Travis: bump go to 1.13.x
 * support vscode
 * change email address
 * add a simple logo
 * Makefile: use go proxy
 * update dependencies
 * Travis: use go 1.12.x instead of tip
 * build(deps): bump github.com/sirupsen/logrus from 1.4.0 to 1.4.2
 * regex: fix escpaces
 * add `make .install.tools` to install dependencies for CI
 * make check: add golangci-lint
 * regex: use raw strings
 * switch to go mod for dependency management

Signed-off-by: Valentin Rothberg <rothberg@redhat.com>
